test/widget_tests/widgets/custom_notification_test.dart

```diff
@@ -0,0 +1,44 @@
+import 'dart:io';
+
+import 'package:firebase_core/firebase_core.dart';
```

Comment:
delete spacing

Reply:
I think the test does not need too much attention to the import and spacing 😅 I will notice it more